qiskit/algorithms/gradients/finite_diff_estimator_gradient.py

```diff
@@ -90,8 +129,15 @@ def _run(
 
         gradients = []
         for result in results:
-            n = len(result.values) // 2  # is always a multiple of 2
-            gradient_ = (result.values[:n] - result.values[n:]) / (2 * self._epsilon)
-            gradients.append(gradient_)
+            if self._method == "central":
+                n = len(result.values) // 2  # is always a multiple of 2
+                gradient_ = (result.values[:n] - result.values[n:]) / self._epsilon
+                gradients.append(gradient_)
```

Comment:
If you want, you could move this append outside the if statements as it's the same in all cases 🙂